.github/workflows/pr-command.yml

```diff
@@ -74,6 +79,9 @@ jobs:
         username: ${{ github.event.pull_request.user.login }}
         password: ${{ secrets.GHCR_PR_TOKEN }}
       if: github.repository_owner == 'cryostatio'
+    - name: check-status
```

Comment:
What is this workflow for? Seems like it links to the relevant pr command workflow that triggered the comment, would the comment be commented at all if the workflow wasn't successful?

Reply:
Oh, is it to signal the success of the /build_test to test the mergibility?

Reply:
Seems to be printed out the PR discussions. We probably do not need to show this. The action status is shown at the bottom itself.